trytond/ir/cron.py

```diff
@@ -111,7 +116,11 @@ def view_attributes(cls):
                 }),
             ]
 
-    def compute_next_call(self, now):
+    def compute_next_call(self, now=None):
+        if now is None:
+            now = datetime.datetime.now()
+        if not (self.interval_type and self.interval_number):
+            return None
         return (now.replace(tzinfo=tz.UTC).astimezone(tz.SERVER)
             + relativedelta(**{self.interval_type: self.interval_number})
```

Comment:
Je pense que c'est mieux de simplement utiliser un relativedelta vide s'il manque une des variables plutôt que de retourner None.

Reply:
J'suis pas certain de comprendre là. 
Du coup on renvoie juste la variable `now` ?

Reply:
Non car on peut calculer le reste.
Donc je verrai quelque chose comme:

interval_delta = relativedelta()
if self.interval_type and self.interval_number is not None:
   setattr(interval_delta, self.interval_type, self.interval_number)
return (now.replace(...)
    + interval_delta
    + relativedelta(...))